gapis/api/templates/specific_gfx_api.cpp.tmpl

```diff
@@ -133,6 +137,10 @@
     if (stack->isValid()) {
       GAPID_DEBUG({{Template "C++.PrintfCommandCallWithLabel" $}});
 
+      #if TARGET_OS == GAPID_OS_ANDROID
+      ATrace_beginSection("{{$name}}");
```

Comment:
It would be nice to be able to do this in a cross-platform way. 
For now this is fine, but it would be nice to figure out how to avoid duplicating this for Desktop.

Reply:
Agreed.

Reply:
Please create an issue to track this TODO.